docs/source/quark_script.rst

```diff
@@ -245,7 +245,7 @@ objInstance.hookMethod(method, watchArgs, watchBacktrace, watchRet)
 
 - **Description**: Hook the target method with Objection.
 - **params**: 
-    1. method: the tagrget API. (type: str or method instance) 
+    1. method: the target API. (type: str or method instance) 
```

Comment:
This typo also exists in [quark/script/objection.py](https://github.com/quark-engine/quark-engine/blob/7cd21ae238d3b540fc94fbdd79f213abbc1fe8fe/quark/script/objection.py#L69). Could you fix it as well?